enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -133,28 +135,27 @@ msgstr ""
 
 #: enterprise/admin/utils.py:43
 msgid ""
-"Bulk operation failed - no users were linked. Please correct the errors "
-"listed below."
+"Error: Learners could not be added. Correct the following errors."
 msgstr ""
 
 #: enterprise/admin/utils.py:44
-msgid "Either \"Course ID\" or \"Program ID\" can be specified, both were."
+msgid "Either \"Course ID\" or \"Program ID\" can be specified, but both were."
 msgstr ""
 
 #: enterprise/admin/utils.py:45
 #, python-brace-format
-msgid "Enrollment mode {course_mode} not available for course {course_id}."
+msgid "Enrollment track {course_mode} is not available for course {course_id}."
```

Comment:
{ course_mode } in single quotes?  Also not sure if we can reference course title here instead of course {course_id}